### ISSUE TITLE
Bad accessibility

### ISSUE THREAD

**Issue by ahmedthebest31** [vercel/hyper#5078]
I am a blying programmer I use nvda screen reader 
I tried with Hyper, but doesn't give me any respond
Actually my screen reader doesn't read anything like a blank page can you make it accessible please.
A lot of applications filled by electron JS very accessible like Visual Studio code why this doesn't?

  - [ ] Your Hyper.app version is **3.0.2**. Please verify your using the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
  - [ ] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

  ---
  - **Any relevant information from devtools?** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_:

  - **Is the issue reproducible in vanilla Hyper.app?**

## Issue

 - **Hyper version**: undefined "3.0.2"

 - **OS ARCH VERSION:** win32 x64 10.0.19042
 - **Electron:** 3.1.3  **LANG:** undefined
 - **SHELL:** undefined   **TERM:** undefined

  <details>
    <summary><strong> ~/.hyper.js contents</strong></summary>
      <pre>
        <code>
          {
  "updateChannel": "stable",
  "fontSize": 12,
  "fontFamily": "Menlo, \"DejaVu Sans Mono\", Consolas, \"Lucida Console\", monospace",
  "fontWeight": "normal",
  "fontWeightBold": "bold",
  "lineHeight": 1,
  "letterSpacing": 0,
  "cursorColor": "rgba(248,28,229,0.8)",
  "cursorAccentColor": "#000",
  "cursorShape": "BLOCK",
  "cursorBlink": false,
  "foregroundColor": "#fff",
  "backgroundColor": "#000",
  "selectionColor": "rgba(248,28,229,0.3)",
  "borderColor": "#333",
  "css": "",
  "termCSS": "",
  "showHamburgerMenu": "",
  "showWindowControls": "",
  "padding": "12px 14px",
  "colors": {
    "black": "#000000",
    "red": "#C51E14",
    "green": "#1DC121",
    "yellow": "#C7C329",
    "blue": "#0A2FC4",
    "magenta": "#C839C5",
    "cyan": "#20C5C6",
    "white": "#C7C7C7",
    "lightBlack": "#686868",
    "lightRed": "#FD6F6B",
    "lightGreen": "#67F86F",
    "lightYellow": "#FFFA72",
    "lightBlue": "#6A76FB",
    "lightMagenta": "#FD7CFC",
    "lightCyan": "#68FDFE",
    "lightWhite": "#FFFFFF"
  },
  "shell": "",
  "shellArgs": [
    "--login"
  ],
  "env": {},
  "bell": "SOUND",
  "copyOnSelect": false,
  "defaultSSHApp": true,
  "quickEdit": false,
  "macOptionSelectionMode": "vertical",
  "webGLRenderer": true
}

          {
  "plugins": [],
  "localPlugins": []
}
        </code>
      </pre>
  </details>

**Comment by LabhanshAgrawal** [comment]
Can you try out the build from https://github.com/LabhanshAgrawal/hyper/actions/runs/381127746
and see if it's working for you
if it does then I can send a pr with the changes

**Comment by LabhanshAgrawal** [comment]
> Can you try out the build from [LabhanshAgrawal/hyper/actions/runs/381127746](https://github.com/LabhanshAgrawal/hyper/actions/runs/381127746)
> and see if it's working for you
> if it does then I can send a pr with the changes

@ahmedSamy31 ping

**Comment by LabhanshAgrawal** [comment]
Direct link https://github.com/LabhanshAgrawal/hyper/suites/1553426646/artifacts/27791561
